cmd/protoc-gen-gorums/dev/zorums_client_gorums.pb.go

```diff
@@ -0,0 +1,67 @@
+// Code generated by protoc-gen-gorums. DO NOT EDIT.
+// versions:
+// 	protoc-gen-gorums v0.7.0-devel
+// 	protoc            v5.29.2
+// source: zorums.proto
+
+package dev
+
+import (
+	context "context"
+	gorums "github.com/relab/gorums"
+	emptypb "google.golang.org/protobuf/types/known/emptypb"
+)
+
+const (
+	// Verify that this generated code is sufficiently up-to-date.
+	_ = gorums.EnforceVersion(7 - gorums.MinVersion)
+	// Verify that the gorums runtime is sufficiently up-to-date.
+	_ = gorums.EnforceVersion(gorums.MaxVersion - 7)
+)
+
+// ZorumsService is the client-side Configuration API for the ZorumsService Service
+type ZorumsServiceConfigurationClient interface {
```

Comment:
I wonder if we want such a long name... What do you think about just calling it `ZorumsServiceClient` to align with gRPC's naming?

We can still use `ZorumsServiceNodeClient` since that's more of a special case for Gorums.